cmd/edgex-secrets-config/README.md

```diff
@@ -0,0 +1,193 @@
+% edgex-secrets-config-proxy(1) User Manuals edgex-secrets-config-proxy(1)
```

Comment:
Name does match executable. I.e. `edgex-secrets-config-proxy` vs ` edgex-secrets-config` 
I think you need to drop `proxy` off the name through out this README.

Reply:
I am patterning this off of the docker man page, where `docker run` is actually the `docker-run` man page.

```
DOCKER(1)                                                                    JUNE 2014                                                                   DOCKER(1)

NAME
       docker-run - Run a command in a new container

SYNOPSIS
       docker run [-a|--attach[=[]]] [--add-host[=[]]] [--blkio-weight[=[BLKIO-WEIGHT]]] [--blkio-weight-device[=[]]] [--cpu-shares[=0]] [--cap-add[=[]]]
```

Reply:
Ok, got it.